cmd/ocm/create/cluster/cmd.go

```diff
@@ -1519,7 +1529,10 @@ func promptPrivateServiceConnect(fs *pflag.FlagSet) error {
 			return err
 		}
 	}
-
+	if isWif && args.gcpPrivateSvcConnect.SvcAttachmentSubnet == "" {
+		return fmt.Errorf(
+			"flag 'psc-subnet' is required when cluster is 'private' and GCP authentication type is %s", c.AuthenticationWif)
```

Comment:
'psc-subnet' and 'private' should be consts instead of hard-coded values for maintainability purposes. By using consts, the value need only be changed in one location to impact all parts of the code that use it. Hard-coded values that must be identifcal in multiple sections of the code rely on the modified to ensure they are thorough when changing the value, which opens a potential for defects as the number of locations increase.